carbon-design-system/carbon
# fix(Select): storybook defaults cleanup

Closes #13008 

#### Changelog

**New**

- Link select stories to mdx docs
- Add guidance in docs to default options

**Changed**

- Remove default "Choose an option" option
- Add empty default option
- Removed categories from stories


#### Testing / Reviewing

Check `Select` stories in deployment:
- Make sure all have an empty default option that shows up on selection when select is opened
- Make sure all of them no longer have categories
- Click on Docs tab an read through new guidance, make sure it's coherent and makes sense.
